source/NVDAObjects/__init__.py

```diff
@@ -1267,3 +1268,35 @@ def getSelectedItemsCount(self,maxCount=2):
 		For performance, this method will only count up to the given maxCount number, and if there is one more above that, then sys.maxint is returned stating that many items are selected.
 		"""
 		return 0
+
+	reviewBounded = False
+	
+	@script(gesture="kb:NVDA+o",
+		# Hardcoded to avoid circular import
+		category=_(u"Text review"),

```

Comment:
Rather than hardcoding here, could this be the right moment to create a new scriptCategories module that contains just script category constants?

Reply:
Might that break add-ons?

Reply:
Yes, but from globalCommands for example, you can do something like from scriptCategories import *